leader.go

```diff
@@ -190,7 +191,7 @@ WATCH_NODES_WAIT:
 
 		// Log a message when the balancing changes.
 		if !reflect.DeepEqual(healthNodes, prevHealthNodes) || !reflect.DeepEqual(pingNodes, prevPingNodes) {
-			a.logger.Printf("[INFO] Rebalanced %d external nodes across %d ESM instances", len(externalNodes), len(healthyInstances))
+			a.logger.Info("", "Rebalanced ", hclog.Fmt("%d external nodes across %d ESM instances", len(externalNodes), len(healthyInstances)))
```

Comment:
There is an extra `""` in the first argument in Info(). It looks like this isn't needed and can be removed. The space after "Rebalanced " can also be removed.